src/content/release/breaking-changes/deprecate-focusable.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Deprecate  `SemanticsProperties.focusable` and `SemanticsConfiguration.isFocusable`
+description: >
+  The `focusable` parameter has been replaced by `isFocused`.
+---
+
+## Summary
+
+The `SemanticsProperties.focusable` and `SemanticsConfiguration.isFocusable` parameters were deprecated
+in favor of the `SemanticsProperties.focused` and `SemanticsConfiguration.isFocused` parameter.
+
+## Context
+
+The `SemanticsConfiguration.isFocusable` is a boolean to describe if the semantics node can have input focus, and `SemanticsConfiguration.isFocused` is a boolean to describe if the semantics node has input focus.
+
+same with `SemanticsProperties.focusable` and `SemanticsProperties.focused`.
+
+## Description of change
+
+The `SemanticsConfiguration.isFocusable` is deprecated in
+favor of `SemanticsConfiguration.isFocused`, which is a nullable boolean, setting it to true/false will automatically make isFocusable to `true` and setting it to null will make isFocusable to false.
+
+## Migration guide
+
+Replace `SemanticsConfiguration.isFocusable` with `SemanticsConfiguration.isFocused`
+
+### Example 1: setting isFocused to `true` or `false` will automatically set isFocusable to `true`.
+
+Code before migration:
+
+```dart
+void describeSemanticsConfiguration(SemanticsConfiguration config) {
+  config.isFocusable = true;
+  config.isFocused = true;
+}
+```
+
+Code after migration:
+
+```dart
+void describeSemanticsConfiguration(SemanticsConfiguration config) {
+  config.isFocused = true;
+}
+```
+
+### Example 2: setting isFocused to null will automatically set isFocusable to `false`.
+
+Code before migration:
+
+```dart
+void describeSemanticsConfiguration(SemanticsConfiguration config) {
+  config.isFocusable = false;
+  config.isFocused = false;
+}
+```
+
+Code after migration:
+
+```dart
+void describeSemanticsConfiguration(SemanticsConfiguration config) {
+  config.isFocused = null;
+}
+```
+
+
+## Timeline
+
+Landed in version: Not yet<br>
+In stable release: Not yet
+
```

Comment:
Is this in the main channel yet? If so, when did it land? Please add that info here.

Reply:
the PR is still pending review, will add the info after the PR is landed